lib/awful/ewmh.lua

```diff
@@ -284,7 +279,6 @@ client.connect_signal("request::tag", ewmh.tag)
 client.connect_signal("request::urgent", ewmh.urgent)
 client.connect_signal("request::geometry", ewmh.geometry)
 client.connect_signal("property::border_width", geometry_change)
-client.connect_signal("property::geometry", geometry_change)
```

Comment:
...sounds like the function name no longer matches its job (although I am not quite sure what that job is). Could you clean that up?